Userland/Utilities/winmove.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022, demostanis worlds <demostanis@protonmail.com>
+ * Copyright (c) 2022, the SerenityOS developers.
```

Comment:
Same here. Everywhere else you use "the SerenityOS developers." or "The SerenityOS developers", I would suggest being consistent about this